pyrobosim/pyrobosim/navigation/rrt.py

```diff
@@ -101,6 +101,14 @@ def plan(self, start, goal):
 
         t_start = time.time()
         goal_found = False
+        direct_connect = False
+
+        direct_goal = self.new_node(n_start, n_goal.pose)
+        direct_connect = self.graph.connect(n_start, direct_goal)
+        if direct_connect:
+            self.graph.add(n_goal)
+            goal_found = True
```

Comment:
Can definitely shorten:

```suggestion
        if self.graph.connect(n_start, n_goal):
            self.graph.add(n_goal)
            goal_found = True
```

Also add a comment saying what this block of code does, e.g., If there is a direct path from start to goal, take it.